cli/src/commands/util/gc.rs

```diff
@@ -27,6 +27,10 @@ use crate::ui::Ui;
 ///
 /// To garbage-collect old operations and the commits/objects referenced by
 /// then, run `jj op abandon ..<some old operation>` before `jj util gc`.
+///
+/// At present, jj's garbage collection is not comprehensive. If a commit is
```

Comment:
I think that's the right thing to do so we don't leave dangling pointers, so I wouldn't call it "not comprehensive".  I'd just drop that sentence.

Reply:
I'm not sure everyone will understand the motivation for second sentence on its own. How about something like:

> At present, jj's garbage collector may clean up less space than you'd expect. If a commit...

But, if you prefer, we can just go with the second sentence.

Reply:
Maybe something like this?

> Note that previous versions of a change are reachable via the evolution log and are therefore not garbage-collected.

Reply:
Sounds pretty good. I did edit it slightly, PTAL.

Reply:
I prefer to not say "currently" because it seems to imply that it will change in the future and I'm not sure it will. Maybe we'll even remove the evolution log completely, but I feel like it's better to not say anything about it yet.

Also, I think there's an extra "that" in there.

Reply:
OK, sure. I didn't realize this might be permanent, but if we're not sure, it is better to not say anything.